config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -158,6 +163,183 @@ function download_rhel_kvm_guest_qcow2() {
 
 }
 
+# Function to extract RVPS value from PODVM SE image
+function calculate_rvps_qcow2() {
+    mkdir -p "${CAA_SRC_DIR}"/podvm/rvps_config
+    cp "${CAA_SRC_DIR}"/podvm/output/*.qcow2 "${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export SE_IMG_PATH="${CAA_SRC_DIR}"/podvm/rvps_config/podvm-libvirt.qcow2
+    export RVPS_CONFIG_DIR="${CAA_SRC_DIR}"/podvm/rvps_config
+    rm -rf $RVPS_CONFIG_DIR/se.img
+    rm -rf /mnt/sevm
+    mkdir /mnt/sevm
+    sleep 5
+    modprobe nbd
+
+    if [ $? -eq 0 ]; then
+    	echo "modprobe ran successfully"
+    else
+        error_exit "modprobe command execution Failed!!"
+    fi
+
+    sleep 10
+
+    qemu-nbd -c /dev/nbd3 "${SE_IMG_PATH}" 
+    if [ $? -eq 0 ]; then
+        echo "nbd connection successful.."
+    else
+	    echo "Retrying manual nbd3 creation..."
+	    mknod /dev/nbd3 b 43 96
+	    if [ $? -eq 0 ]; then
+		    echo "device nbd3 created successfully"
+		    qemu-nbd -c /dev/nbd3 "${SE_IMG_PATH}"
+		    if [ $? -eq 0 ]; then
+			    echo "nbd connection successful"
+		    else
+			    echo "nbd connection failed"
+		    fi
+	    else
+		    echo "device nbd3 creation Failed!!"
+	    fi
+    fi
+
+    mknod /dev/nbd3p1 b 43 97
+    if [ $? -eq 0 ]; then
+        echo "nbd3p1 node creation successful.."
+    else
+        echo "nbd3p1 node creation Failed!!"
+    fi
+
+    mount /dev/nbd3p1 /mnt/sevm
+    if [ $? -eq 0 ]; then
+	    echo "mounting for PODVM successful.."
+    else
+	    sleep 20
+	    mount /dev/nbd3p1 /mnt/sevm
+	    if [ $? -eq 0 ]; then
+		    echo "mounting for PODVM successful on attempt 2"
+	    else
+		    echo "mounting for PODVM Failed!!"
+	    fi
+    fi
+
+    cp /mnt/sevm/se.img "${RVPS_CONFIG_DIR}" 
+    if [ $? -eq 0 ]; then
+        echo "copying of se.img successful.."
+    else
+        echo "copying of se.img Failed!!"
+    fi
+
+    umount /mnt/sevm
+    if [ $? -eq 0 ]; then
+        echo "unmounting for PODVM successful.."
+    else
+        echo "unmounting for PODVM Failed!!"
+    fi
+
+    qemu-nbd -d /dev/nbd3
+    rm -rf $RVPS_CONFIG_DIR/hdr.bin
+    git clone https://github.com/ibm-s390-linux/s390-tools.git
```

Comment:
maybe we can add more info on `s390-tools` repo and might be worth checking out to a tag instead of using the main branch?

Reply:
I have added wget and copying only one file rather than copying complete repo.